elixir/apps/api/test/api/gateway/channel_test.exs

```diff
@@ -118,6 +118,17 @@ defmodule API.Gateway.ChannelTest do
                  %{protocol: :icmp}
                ]
              }
+      assert payload.client == %{
+               id: client.id,
+               peer: %{
+                 ipv4: client.ipv4,
+                 ipv6: client.ipv6,
+                 persistent_keepalive: 25,
+                 preshared_key: preshared_key,
```

Comment:
@AndrewDryga I am not sure where I should be taking this `preshared_key` from as it is loaded from the database in the production code. Could you please fix this? :pray:

Reply:
I think this comes from the client? I took a stab at fixing this with the most recent commit. Feel free to revert.

Reply:
Ok pushed